finder-app/writer.c

```diff
@@ -0,0 +1,76 @@
+// Author- Chaithra Suresh
+// C code to replace writer.sh
+// use tail -f /var/log/syslog | grep writer to see results
+
+
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <stdlib.h>
+#include <syslog.h>
+#include <syslog.h>
+#include <string.h>
+#include <fcntl.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[])
+{
+	
+	
+	if( argc < 3 ) 
+	{
+	
+     		 syslog(LOG_USER,"Too less arguments\n Expected arguments: 1. path to write 2. string to write \n");
```

Comment:
@chaisuresh these prints, since they are interactive, should probably go to standard out.